cogeo_mosaic/cache.py

```diff
@@ -0,0 +1,39 @@
+"""backend cache configuration"""
+import functools
+import os
+from typing import Callable
+
+from cachetools import TTLCache, cached
+
+# TTL of the cache in seconds
+CACHE_TTL = int(os.getenv("CACHE_TTL", 300))
+
+# Maximum size of the LRU cache in MB
+CACHE_MAXSIZE = int(os.getenv("CACHE_MAXSIZE", 512))
```

Comment:
Maybe these env variables should be prefixed with `COGEO_MOSAIC`

Reply:
👍